airflow/models.py

```diff
@@ -1396,7 +1399,10 @@ def signal_handler(signum, frame):
         except (Exception, KeyboardInterrupt) as e:
             self.handle_failure(e, test_mode, context)
             raise
-
+		finally:
+			# After execution, unset these env vars
+			os.environ.pop('CLOVER_DAG_ID', '')
+            os.environ.pop('CLOVER_TASK_ID', '')
```

Comment:
@rohang06 is this a typo with the tab spacing?

Reply:
Good catch, fixed in next diff